### ISSUE TITLE
Integrate to GlassFish

### ISSUE THREAD

**Issue by m0mus** [eclipse-ee4j/servlet-api#222]
- [ ] Integrate to Eclipse GlassFish
- [ ] Pass all CTS/TCK tests

#212 must be finished before starting working on this task.

**Comment by vinayvishal** [comment]
I see this is already done: https://github.com/eclipse-ee4j/glassfish/pull/22615
